database/changesets/20210422_CreateSourceTable.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
Can this file be deleted? (Broken record, sorry…)

Reply:
Deleted (no problem)